DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/src/ForwardSubstitution.hpp

```diff
@@ -135,7 +135,7 @@ event SubmitForwardSubstitutionKernel(queue& q) {
         }  // end of for(i...)
 
         // Loop through all the y vectors
-        for (char vector_num = 0; vector_num < (char)k_num_y_vectors;
+        for (unsigned char vector_num = 0; vector_num < (char)k_num_y_vectors;
```

Comment:
The type of the variables should match.     unsigned char != char

Reply:
Good catch.  Fixed.